plan/new.go

```diff
@@ -52,7 +52,23 @@ func buildTestCases(plan *Plan) []TestCase {
 				Plan:      plan,
 				Client:    client,
 				Arguments: testArgs,
+				Skip:      false,
 			}
+
+			for _, filter := range behavior.SkipFilters {
+				// Each filter should be a complete match
+				isCompleteMatch := true
```

Comment:
I like the readability more. Changed it to that. However note that this tries to match every key value in the map and is less efficient.